client/lib/mixins/i18n/index.js

```diff
@@ -14,7 +14,7 @@ var debug = require( 'debug' )( 'calypso:i18n' ),
 var config = require( 'config' ),
 	numberFormatPHPJS = require( './number-format' ),
 	emitter = require( 'lib/mixins/emitter' ),
-	interpolateComponents = require( 'lib/interpolate-components' );
+	interpolateComponents = require( 'lib/interpolate-components/src/' );
```

Comment:
>  The package.json already has a main property that is set to lib/index.js which is created once it is compiled.

The `package.json` for `interpolate-components` in this PR has "main" set to `src/index.js`. Though I agree that we would want it set to the generated `lib/index.js` so the module works correctly for people requiring it through npm.